src/components/selects/MonthSelect.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+// COMPONENTS
+import Select from "./Select";
+
+const MONTH_OPTIONS = [
```

Comment:
The months to be displayed should be passed in as a prop, and `MONTH_OPTIONS` should be the default value of that prop. Passing in the months as a prop would make it possible to use the component in different scenarios.